src/view.ts

```diff
@@ -0,0 +1,62 @@
+import { Graph, Vertex } from "./graph.ts";
```

Comment:
I've been trying to get into the habit of importing types separately so that it only becomes a compile time dependency. Doesn't really affect anything in this case though. @dagda1 isn't there some linting rule for this I saw you using at one point?

```suggestion
import type { Graph, Vertex } from "./graph.ts";
```